ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -11444,23 +11440,25 @@ static int zend_jit_switch(dasm_State **Dst, const zend_op *opline, const zend_o
 					|	cmp FCARG2a, jumptable->nNumUsed
 					|	jae >3
 					|.if X64
-					|	// TODO: DynASM stores .aword as 4-bytes even in 64-bit mode ???
-					|	movsxd r0, dword [FCARG2a * 4 + >4]
-					|	jmp r0
+						if (!IS_32BIT(dasm_end)) {
+							|	lea r0, aword [>4]
```

Comment:
Thank you, I was not aware of this pattern.

Reply:
@nikic it is mentioned at https://corsix.github.io/dynasm-doc/instructions.html#memory